apps/events.md

```diff
@@ -127,105 +150,448 @@ cart: {
     subAmount: "76,01 €"
     taxType: "NET"
     taxes: []
+    totalBasketDiscount: null
     totalNumberOfItems: 3
     _links: null
 }
 ```
-## Category event
 
-To make use of this event, you need to add the following snippet to your code:
+### product:click
+
+To get more information about this event, you can use the following snippet:
+
+```js
+if (window.eComEventTarget) {
+  window.eComEventTarget.addEventListener('product:click', function (event) {
+    console.log('product:click', {
+      type: event.detail.type,
+      detail: event.detail.detail,
+      product: event.detail.product,
+      productIndex: event.detail.productIndex,
+    })
+  })
+}
+```
+
+Here's an example of what you can get:
+
+```
+detail: "Cherry Jam"
+product: {
+    availabilityText: "Available"
+    available: true
+    basePrice: {refQuantity: {…}, refPrice: {…}, formatted: "1 m³ = €0,12", quantity: {…}}
+    bulkPrices: null
+    conditionMicrodata: "NewCondition"
+    customAttributes: [{…}]
+    deliveryPeriod: "2-3"
+    deliveryPeriodUnit: "DAYS"
+    description: null
+    energyLabel: null
+    energyLabelSourceFile: null
+    gtin: 7501054530107
+    hasCrossSelling: false
+    hasStockLevel: true
+    hasVariations: false
+    href: "/p/homemade-cherry-jam"
+    image: null
+    isVariationMaster: false
+    isVariationProduct: false
+    isVisible: true
+    links: (5) [{…}, {…}, {…}, {…}, {…}]
+    listPrice: null
+    lowestPrice: null
+    mainCategoryId: "5954B711-E377-2A90-C400-D5809AB3F62B"
+    manufacturer: null
+    manufacturerPrice: null
+    metaDescription: ""
+    name: "Homemade Cherry Jam"
+    onStock: true
+    outOfStock: false
+    pickupPeriod: ""
+    price: {taxType: "NET", formatted: "€40,00", amount: 40, currency: "EUR"}
+    productDataSheet: null
+    productId: "5954B706-E701-F357-A52D-D5809AB3F606"
+    productVariationSelection: null
+    productVariationValues: ""
+    sku: "1007"
+    slideshow: [{…}, {…}, {…}, {…}]
+    slug: "homemade-cherry-jam"
+    stockLevelClass: "in"
+    stockLevelMicrodata: "InStock"
+    title: "Homemade Cherry Jam"
+    variationMaster: null
+    variations: null
+    vatNote: "components.productComponent.priceExclusiveVat"
+    warnStock: false
+    weight: null
+}
+productIndex: 1
+type: "search"
+```
+
+### wishlist:add
+
+To get more information about this event, you can use the following snippet:
+
+```js
+if (window.eComEventTarget) {
+  window.eComEventTarget.addEventListener('wishlist:add', function (event) {
+    console.log('wishlist:add', {
+      locale: event.detail.locale,
+      productId: event.detail.product.productId,
+      quantity: event.detail.product.quantity,
+      lineItemId: event.detail.lineItemId,
+    })
+  })
+}
+```
+
+Here's an example of what you can get:
+
+```
+lineItemId: undefined
```

Comment:
Can we use a better example, or was it always `undefined` when you tried?

Reply:
It doesn‘t look normal, but it is the same for me when I tried it out. I just created a bug ticket.

For now, I suggest to remove documentation for this property.